akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -44,6 +44,8 @@ object ClusterSingletonManagerSettings {
    */
   def apply(system: ActorSystem): ClusterSingletonManagerSettings =
     apply(system.settings.config.getConfig("akka.cluster.singleton"))
+      // note that this setting has some additional logic inside the ClusterSingletonManager
+      // falling back to DowningProvider.downRemovalMargin if it is off/Zero
       .withRemovalMargin(Cluster(system).settings.DownRemovalMargin)
```

Comment:
there is a deprecated annotation at `DownRemovalMargin` also

Reply:
I think it will be the same result if we change this to `Cluster(system).downingProvider.downRemovalMargin`.

We would still have to keep the zero check in ClusterSingletonManager for the case `apply(config: Config)`.

Reply:
I left that other deprecation on purpose, I thought that the config property is ok but if any other other third party or future cluster tool thinks they should use it, they shouldn't.

For the `Cluster(system).downingProvider.downRemovalMargin`, if someone is running an older version of SBR that would behave differently, since that doesn't have the fallback part and would choose `stable-after` instead, that's why I left it for now, with a comment, here.

Reply:
Ok, good